app/views/content_items/fatality_notice.html.erb

```diff
@@ -10,6 +9,7 @@
 </div>
 
 <%= render 'shared/publisher_metadata_with_logo' %>
+<%= render 'components/notice', @content_item.withdrawal_notice_component  %>
```

Comment:
@miaallers Is the notice component meant to be full width or 2/3rds? I think this will render it at full width.

Reply:
It does render full width, which is how it looks in Alex's ordering diagram.  Will run it past Mia.

Reply:
Yep!